### ISSUE TITLE
Assigning "splash" when using team_datasets

### ISSUE THREAD

**Issue by Ligerithm** [pmariglia/showdown#167]
This AI is pretty remarkable and contains comprehensive battle simulation and a nice state analysis function.

But even when I use team_datasets AI, when a set for a character isn't found, it assumes that the enemy only knows splash. I was under the assumption that if there wasn't a hard-coded set, it would just assume an ability, item, and moveset based on current usage data. There appears to be a smogon usage parser, so I'm not sure what i'm doing wrong.

```
[WARNING]  No sets for miraidon, trying to find most likely attributes
[WARNING]  miraidon not in the sets lookup, using random battle abilities
[WARNING]  miraidon not in the random-battle sets lookup
[WARNING]  miraidon not in the sets lookup, using random battle items
[WARNING]  miraidon not in the random-battle sets lookup
[WARNING]  miraidon not in the sets lookup
[WARNING]  miraidon not in the random-battle sets lookup
[WARNING]  miraidon not in the sets lookup
splash: 100%
Move: freezedry, Prominence: 424.0
Move: switch lilligant, Prominence: 420.0
Move: switch ironmoth, Prominence: 420.0
Move: switch walkingwake, Prominence: 420.0
Move: switch basculegionf, Prominence: 420.0
Move: roost, Prominence: 404.0
Move: substitute, Prominence: 400.0
Move: hurricane, Prominence: 394.1
[DEBUG]    Most Prominent: freezedry
```

This apparent blindness caused my Articuno to believe the Miraidon only had Splash, which unfortunately cost its life. There seem to be many similar instances of this.

I have a custom move selection function, yes, but that's all I changed. The actual sets lookup code is identical to team datasets.

Basically, what I'm wondering is: Is there a function that, when sets lookups fails, can parse smogon data to come up with the most probable set for the monster? Even if it's just blindly choosing the top 4 moves and top item, it's a far better premonition to have than "splash."

The initial appraisal could obviously get more nuanced and have re-appraising methods like adjusting predicted items, EVs, abilities, and moves when Choice Band damage is seen, a certain amount of bulk is implicated, leftovers healing is/isn't seen, etc. I intend to attempt such functionality, but right now I really just seek a way to get some rudimentary usage data added to the set. Wondering if there's any code for it, and why it's not working.

**Comment by Ligerithm** [comment]
(in random battles)

**Comment by Ligerithm** [comment]
on further reflection, it seems this is an issue exclusive to random battles. actual OU is pretty precise, but there's no usage data for randbats it appears and so set prediction must be done differently
